### PR TITLE
Astghik

### PR DESCRIPTION
**Pull request by AstghHarutyunyan**

### CODE REVIEW

**Comment by AstghHarutyunyan** [comment]
test commit

File: Task_3.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Write bash command(s) to read content of "test.txt" file, change all "test" keywords with "DONE" keyword and write in "test.result.txt" file.
+
+
+1. Open terminal
+2. Go to the corresponding place -> cd Desktop/pdp_training command
+3. Enter while read line; do echo $line; done < test.txt command
+4. Enter sed 's/testing/DONE/g; /test/DONE/g; s/Test/DONE/g' test.txt > test.result.txt command
+
```
Comment (arayik-chaparyan) [inline Task_3.txt]: Astghik jan 
 - executed command from line 3 and it shows content for test.txt file
 - executed command from line 4 and it fails with error 'sed: 1: "s/testing/DONE/g; /test ...": extra characters at the end of D command'

Could you please remove all extra words from solution and keep just solution in 'Task_3.txt' file.

**Comment by AstghHarutyunyan** [comment]
Arayik jan I have updated Task_3

**Comment by AstghHarutyunyan** [comment]
added the 1st task

File: Task_1.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+https://www.imdb.com/find?s=kw&q={{keyword}}&ref_=nv_sr_sm
+
+3. Run the collection using .CSV file (upload the file before start to run)
+
+4. Check the result status using the following test script: 
+
+pm.test("Status code is 200", function () {
+    pm.response.to.have.status(200);
+});
```
Comment (arayik-chaparyan) [inline Task_1.txt:13]: Astghik jan thank you. Overall looks good.
Postmap has an ability to export and import collections for easy sharing.

**Comment by arayik-chaparyan** [comment]
Astghik jan thank you for all results. Overall looks good.
I'm closing the PR.

**Pull request closed without merging**